sensors/triggers/standard-k8s/starndard-k8s.go

```diff
@@ -114,22 +116,36 @@ func (k8sTrigger *StandardK8sTrigger) Execute(resource interface{}) (interface{}
 	}
 	obj.SetNamespace(namespace)
 
-	op := v1alpha1.Create
-	if trigger.Template.K8s.Operation != "" {
+	// If object not found, create the object. No need to have a separate `Create` switch case.
+	oldObj, err := k8sTrigger.namespableDynamicClient.Namespace(namespace).Get(obj.GetName(), metav1.GetOptions{})
+	if err != nil {
+		if !apierr.IsNotFound(err) {
+			return nil, err
+		}
+		return k8sTrigger.namespableDynamicClient.Namespace(namespace).Create(obj, metav1.CreateOptions{})
```

Comment:
When `op==Patch && oldObj not existing`, it also does a `create`, this looks weird.

Reply:
Similar to `kubectl apply`. Most of the users expect the sensor to create the object if not available or update/patch the object otherwise. The user who raised the issue also have similar requirements.

Reply:
I agree with that. I thought the `patch` option had different `resource` than `update` and `create`, just like `kubectl patch deployment xxxxx --patch patch_file`, which `patch_file` is not a valid `deployment` spec.